webpack.config.js

```diff
@@ -72,7 +72,11 @@ module.exports = (env, argv) => {
       warningsFilter: /export .* was not found in/
     },
     plugins: [
-      new ForkTsCheckerWebpackPlugin(),
+      new ForkTsCheckerWebpackPlugin({
```

Comment:
What's this all do?

Reply:
The build was hanging/failing in ForkTsCheckerWebpackPlugin which I eventually tracked down to the user of the TypeScript incremental API, which is supposed to speed up builds by caching information from one build to the next and only checking changed files incrementally. But reports vary on whether it actually speeds up the build and a successful build is better than one that fails or never completes. The other option allows the type-checker to utilize multiple cores which it can't do when using the incremental option, but can when it's disabled. So now our type-checking will be done in parallel but not incrementally.